index.js

```diff
@@ -8,7 +8,13 @@
   // Stub out `now` so we can use a more precise number in uuid generation, if
   // available.
   function now() {
-    return global.performance ? global.performance.now() : (new Date()).getTime();
+    if (global.performance && global.performance.now instanceof Function) {
```

Comment:
Force of habit – I tend to use `instanceof` for everything (except primitive types it doesn't work with) since `typeof` doesn't work well for complex / custom objects (everything's an `object`).  I've also been under the impression that `instanceof` checks are slightly faster (not really important here) but that no longer appears to be true.  Doesn't really matter to me either way.